paddle/fluid/operators/batch_norm_op.cu

```diff
@@ -46,6 +46,7 @@ class BatchNormKernel<platform::CUDADeviceContext, T>
     float momentum = ctx.Attr<float>("momentum");
     const bool is_test = ctx.Attr<bool>("is_test");
     const bool use_global_stats = ctx.Attr<bool>("use_global_stats");
+    const bool fuse_with_relu = ctx.Attr<bool>("fuse_with_relu");
```

Comment:
You can get the supported act_type in `cudnnBatchNormalizationForwardTrainingEx`.
Not only relu.